app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(cp: str) -> None:
+    command, file_in, file_out = cp.split()
+    if command == "cp":
```

Comment:
Add a condition to check if `file_in` and `file_out` are the same. According to the task description, the function should do nothing if the source and destination file names are identical.